Bottom: af42a5fbaabc8e4b42819d47db53b9fe48551757
Top:    f806455fe96c86b67081a98452fe05f8e90ce189
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-22 22:40:40 +0100

Fix last commit to work with new crosshair notification API


---

diff --git a/src/action.c b/src/action.c
index 7adb9de..4181f2e 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4847,7 +4847,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
   char *function = ARG (0);
   if (function && PCB->ElementOn)
     {
-      HideCrosshair (true);
       switch (GetFunctionID (function))
 	{
 	case F_Object:
@@ -4897,7 +4896,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 	      }
 	  }
 	}
-      RestoreCrosshair (true);
     }
   return 0;
 }
